test/unit/rustdoc/rustdoc_unit_test.bzl

```diff
@@ -90,6 +90,7 @@ def _target_maker(rule_fn, name, rustdoc_deps = [], **kwargs):
     rust_doc(
         name = "{}_doc".format(name),
         crate = ":{}".format(name),
+        crate_features = kwargs.get("crate_features", []),
```

Comment:
I'd also like to confirm that the crate features value ends up on the command line. I believe we can do that by putting a specific feature here and checking that it is contained in the command line here:https://github.com/bazelbuild/rules_rust/blob/7bbe005ca8e4747d6c48cd73985ae47424eb8a2e/test/unit/rustdoc/rustdoc_unit_test.bzl#L12